src/main/java/seedu/address/model/person/NameContainsKeywordsPredicate.java

```diff
@@ -41,4 +41,8 @@ public boolean equals(Object other) {
     public String toString() {
         return new ToStringBuilder(this).add("keywords", keywords).toString();
     }
+
+    public boolean isEmpty() {
+        return keywords.contains("");
```

Comment:
Should this be `equals("")` instead of `contains("")`?

Reply:
Because keywords is a List<String> and not just a String hence .equals("") would not work here.